diamond_agent/tasks.py

```diff
@@ -220,8 +220,21 @@ def config_handlers(ctx, handlers, config_path, handlers_path):
     """
     if handlers is None:
         handlers = copy_objects.deepcopy(DEFAULT_HANDLERS)
-        handlers['cloudify_handler.cloudify.CloudifyHandler']['config'][
-            'server'] = get_manager_ip()
+
+        cloudify_agent = ctx.bootstrap_context.cloudify_agent
```

Comment:
you cannot rely on this configuration existing. you should support the case where cloudify_agent is not configured in provider context and fallback to default `guest:guest`. there should be no error raised here